### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by caipre**
These changes add some logging information that was useful to me when working with clair.

### CODE REVIEW

**Review by jzelinskie — APPROVED** [review]
LGTM. Thanks!

File: worker.go [hunk 1]
```diff
@@ -210,7 +210,7 @@ func detectFeatureVersions(name string, files tarutil.FilesMap, namespace *datab
 			continue
 		}
 
-		log.Warningf("layer %s: Layer's namespace is unknown but non-namespaced features have been detected", name)
+		log.Warningf("Namespace unknown for feature %s %s, in layer %s", feature.Feature.Name, feature.Version, name)
```
Comment (Quentin-M) [inline worker.go:213]: why removing the prefix? the format is used everywhere else.

Comment (Quentin-M) [reply worker.go:213]: ideally, we'd like to switch to logrus and use structured messages though!

Comment (jzelinskie) [reply worker.go:213]: I'm going to make a follow up PR doing that.

**Pull request merged**